tasks/fix-cat2.yml

```diff
@@ -1603,10 +1615,15 @@
       - name: "MEDIUM | RHEL-07-032010 | AUDIT | The system must update the DoD-approved virus scan program every seven days or more frequently."
         debug:
             msg: "{{ rhel_07_032010_audit.stdout_lines }}"
+        changed_when:
+            - rhel7stig_audit_complex
```

Comment:
Should there be a conditional here to only show changed when rhel_07_032010_audit.stdout_lines contains something?

Reply:
likewise